version_test.go

```diff
@@ -33,10 +37,12 @@ func (s *VersionSuite) TestVersionExtraArgs(c *gc.C) {
 		Stdout: &stdout,
 		Stderr: &stderr,
 	}
+	setTestLoggingBuffer(c, &stderr)
+
 	code := Main(newVersionCommand("xxx"), ctx, []string{"foo"})
```

Comment:
@jameinel without this change the test won't function as intended since the test's faked standard error would not be the global logger's underlying `io.writer`.